zerver/openapi/zulip.yaml

```diff
@@ -678,7 +678,7 @@ paths:
                                   "subscriptions":
                                     [
                                       {
-                                        "name": "test_stream",
+                                        "name": "test",
```

Comment:
I'm not sure I like either `test_stream` or `test` for this sort of thing - but `test_channel` would be better for just making clear this is expected to be a channel name.